core/src/main/java/jenkins/install/SetupWizard.java

```diff
@@ -113,6 +115,19 @@ public SetupWizard(Jenkins j) throws IOException {
      */
     public HttpResponse doCompleteInstall(StaplerRequest req, StaplerResponse rsp) throws IOException, ServletException {
         Jenkins j = Jenkins.getActiveInstance();
+        User u = j.getUser("admin");
+        // JENKINS-33572 - without creating a new 'admin' user, auth key erroneously remained
+        if(u != null && u.getProperty(AuthenticationKey.class) != null) {
+            // There must be a better way of removing things...
```

Comment:
Evidently not; for historical reasons, this is not a `DescribableList`. You could add a `removeProperty` method.